tripy/CONTRIBUTING.md

```diff
@@ -31,6 +31,8 @@ Thank you for considering contributing to Tripy!
         docker run --gpus all -it -v $(pwd):/tripy/ --rm ghcr.io/nvidia/tensorrt-incubator/tripy
         ```
 
+        If you have pulled the image in the past and need to update it, you should add `--pull=always` to the above command or separately call `docker pull ghcr.io/nvidia/tensorrt-incubator/tripy`.
```

Comment:
Let's just make this the default and add `--pull=always` to the code block above? If an older container was required, we'd be building it locally in most cases.

Reply:
Good idea. Also the equals sign is wrong, so I'll take that out.